src/routes/userRoutes.js

```diff
@@ -35,5 +35,33 @@ routes.resetPassword = async (req, res) => {
   }
 }
 
+routes.updateDBPassword = async(req,res) =>{
+    if(req.params.id===null ||!req.body.password ){
+      logger.info('invalid input')
+      return res.status(400).json({error:{message:'invalid input of userid and password'}})
+  }
+  const {Accounts} = db.getModels()
+      const userAccount = await Accounts.findOne({
+          where:{
+              id:req.params.id
+          }
+      })
+      if(!userAccount){
+        logger.info(`account does not exist`)
+        res.status(400).json({error:{message:"account does not exist"}})
+        return
+    }
+  try {
+
+    await setDBPassword(userAccount,req.body.password) 
+    logger.info(`user ${userAccount.id} updates password`)
+    return res.status(200).json('success')
```

Comment:
Return an object to be consistent with your other json calls

```
return res.status(200).json({message: "success"})

Reply:
i guess just success is fine as long as it's json. plz check the exercise in js5 for reference:)